lib/names/workloadendpoint.go

```diff
@@ -143,13 +143,6 @@ func (ids WorkloadEndpointIdentifiers) getSegments() ([]segment, error) {
 			},
 		}
 	}
-	if len(orch.value) == 0 {
```

Comment:
Why were we requiring this before? @robbrockbank?

Also, the comment on line 138 needs updating to match this change.

Reply:
I don't think I implemented this correctly. I think we can pass when _just_ a node is specified, but we should error when all identifiers are provided besides orchestrator. right?